setup.py

```diff
@@ -49,7 +49,7 @@ def build_extensions(self):
     url='https://github.com/nucleic/kiwi',
     description='A fast implementation of the Cassowary constraint solver',
     long_description=open('README.rst').read(),
-    install_requires=['distribute'],
+    install_requires=['setuptools'],
```

Comment:
`install_requires` is for [runtime deps](https://packaging.python.org/discussions/install-requires-vs-requirements/#install-requires). Since this is a shared library, is `setuptools` really required at runtime?